joomla/joomla-cms
# [4.0] left sidebar align

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/26201.

### Summary of Changes
No margin for sidebar left.


### Testing Instructions

1. Apply this patch and run `node build.js --compile-css` for updating the changed `scss` file. Alternatively you can run `npm i`.
2. Add a module to the left sidebar.
3. Open the side in the front end and see, that the Cassiopeia sidebars and component area horizontally aligned.



### Expected result
Cassiopeia sidebars and component area horizontally aligned.


### Actual result
Cassiopeia sidebars and component area not horizontally aligned.


### Documentation Changes Required
No
